src/components/Admin/index.jsx

```diff
@@ -230,6 +241,7 @@ class Admin extends React.Component {
                   </div>
                   <div className="col-12 col-md-6 text-md-right">
                     {this.renderDownloadCsvButton()}
```

Comment:
We should remove `renderDownloadCsvButton` and the call to it here now that it's in its own component.